.github/workflows/build-citus-community-nightlies.yml

```diff
@@ -31,6 +31,7 @@ jobs:
           - debian/stretch
           - debian/buster
           - debian/bullseye
+          - debian/bullseye
```

Comment:
why we added one more debian/bullseye in 1b4558a2183851e3d8e13f5184dd1133ddad56c5 ?

Reply:
my mistake removed it